provider/cmd/pulumi-gen-xyz/main.go

```diff
@@ -34,7 +34,7 @@ import (
 // TemplateDir is the path to the base directory for code generator templates.
 var TemplateDir string
 
-// BaseDir is the path to the base pulumi-kubernetes directory.
+// BaseDir is the path to the base pulumi-provider directory.
```

Comment:
I'd change this to `pulumi-xyz` so it's easier to update with a find/replace.

Reply:
Done!